src/wxProfilerGUI/profilergui.cpp

```diff
@@ -637,6 +639,26 @@ bool ProfilerGUI::Run()
 	else if (!cmdline_attach.empty())
 	{
 		std::unique_ptr<AttachInfo> info(AttachToProcess(cmdline_attach));
+		if (!cmdline_thread_ids.empty()) {
+			std::cout << "(Orig) Profiling Threads: ";

```

Comment:
I'm not sure of the value of leaving these debug messages here.

Reply:
You're correct. I thought I'd removed these.

Reply:
I see now that they've been removed in follow-up commits. In this case please just squash the removal with the addition.